Interop/ModelCard/src/main/java/org/tribuo/interop/modelcard/UsageDetailsBuilder.java

```diff
@@ -0,0 +1,146 @@
+package org.tribuo.interop.modelcard;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+
+public class UsageDetailsBuilder {
+    private String intendedUse = "";
+    private String intendedUsers = "";
+    private final List<String> outOfScopeUses = new ArrayList<>();
+    private final List<String> preProcessingSteps = new ArrayList<>();
+    private final List<String> considerations = new ArrayList<>();
+    private final List<String> factors = new ArrayList<>();
+    private final List<String> resources = new ArrayList<>();
+    private String primaryContact = "";
+    private String modelCitation = "";
+    private String modelLicense = "";
+
+    public String getIntendedUse() {
```

Comment:
I think it's better to explicitly add the no-args constructor rather than have `javac` silently add it for you, as otherwise it gets quite confusing when you add another constructor, the generated no-args one disappears and things don't compile.